Add whisper endpoints (#200)
alexrudall/ruby-openai: CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [3.5.0] - 2023-03-02

### Added

> - Add Client#transcribe endpoint - Whisper transcription over the wire!
> - Add Client#translate endpoint - Whisper translation over the wire!

Reviewer: ```suggestion
- Add Client#transcribe and Client translate endpoints - Whisper over the wire! Thanks to [@Clemalfroy](https://github.com/Clemalfroy)

```
